src/OrbitGl/TimeGraphLayout.cpp

```diff
@@ -26,7 +26,7 @@ TimeGraphLayout::TimeGraphLayout() {
   track_label_offset_x_ = 30.f;
   slider_width_ = 15.f;
   track_tab_width_ = 350.f;
-  track_tab_height_ = 30.f;
+  track_tab_height_ = 25.f;
```

Comment:
Why after this change everything seems to be similar?

Reply:
I've changed the way the `TrackTopMargin` (now called `TrackContentTopMargin`) works because after talking to @florian-kuebler, we've decided it's inconsitent.

Before, the margin was subtracted from the tab height when drawing the tabs. Now, the margin is added inside the track after the complete tab height. Since this margin is 5, I reduced the tab height accordingly so the result is the same.